tests/test_utils/test_assigner.py

```diff
@@ -628,3 +628,22 @@ def test_mask_hungarian_match_assigner():
         dice_cost=dict(type='DiceCost', weight=0.0, pred_act=True, eps=1.0))
     with pytest.raises(AssertionError):
         self = MaskHungarianAssigner(**assigner_cfg)
+
+    # test with solo_style mode of DiceLossCost which is not supported yet
+    assigner_cfg = dict(
```

Comment:
The unit tests do not really test the module to be added in this PR.

Reply:
yes, it was modified, I will fix it.

> The unit tests do not really test the module to be added in this PR.
yes, it was modified, I will fix it.